### ISSUE TITLE
Sign modded files

### ISSUE THREAD

**Issue by Vichingo455** [MishaProductions/Rectify11Installer#136]
You can use signer from endermanch to do that, just use this command:
`Signer /sign sign file /machine`
Should work

**Comment by Vichingo455** [comment]
As well installing the self signed certificate for the local machine will fix the red uac, as the Windows signature uses that way

**Comment by Vichingo455** [comment]
I mean in Trusted CAs

**Comment by HolyNetworkAdapter** [comment]
Sorry, we're not begging microsoft to give us their cert, and we're not paying a fortune to get our own either.

**Comment by Vichingo455** [comment]
> Sorry, we're not begging microsoft to give us their cert, and we're not paying a fortune to get our own either.

You didn't even understand what I typed. I said using the Signer enderman used to self-sign the files, so you can probably avoid the red uac, since the certificate should be installed in the Trusted CAs

**Comment by Typical0** [comment]
Please understand that using something like this will cause more legitimacy issues than v2.5 ISO.

**Comment by ojask** [comment]
@Vichingo455 yeah it sounds like a good idea, we will probably have to add a loop for signer to sign the files before theyre copied over, thank you.

**Comment by ojask** [comment]
@Vichingo455 also, from where can I download Signer?

**Comment by Vichingo455** [comment]
From Endermanch: https://dl.malwarewatch.org/software/advanced/Signer.zip
Password: mysubsarethebest

**Comment by Lixkote** [comment]
> From Endermanch: https://dl.malwarewatch.org/software/advanced/Signer.zip Password: mysubsarethebest

ok

**Comment by MishaProductions** [comment]
Implemented in commit [d5e02f4](https://github.com/MishaProductions/Rectify11Installer/commit/d5e02f4e6622b67c9f6517c26e3773545723b92f)

**Comment by xverge** [comment]
Signing was disabled in 2592f4d, this issue should be reopened.

**Comment by HolyNetworkAdapter** [comment]
We disabled it because it caused too many issues, im not a developer for this but i dont think its coming back anytime soon

**Comment by evansidhu** [comment]
If you really want signed files then just sign them yourself, example: signer.exe /sign “Microsoft Corporation” C:\Windows\System32\cmd.exe
